modules/core/src/main/kotlin/datamaintain/core/step/executor/Executor.kt

```diff
@@ -73,8 +74,13 @@ class Executor(private val context: Context) {
                 logger.info {"start marking script ${script.name} as executed"}
                 val executedScript = ExecutedScript.build(script, Execution(ExecutionStatus.OK), context.config.flags)
 
-                markAsExecuted(executedScript)
-                if (!context.config.porcelain) { logger.info { "${executedScript.name} only marked as executed (so not executed)" } }
+                try {
+                    markAsExecuted(executedScript)
+                    if (!context.config.porcelain) { logger.info { "${executedScript.name} only marked as executed (so not executed)" } }
+                } catch (e: DatamaintainQueryException) {
+                    logger.warn { "Failed to mark script $executedScript as executed. Use the following command to force mark the script as executed : " +
+                            "./datamaintain-cli --db-type ${context.config.driverConfig.dbType} --db-uri ${context.config.driverConfig.uri} update-db --path ${context.config.path} --identifier-regex ${executedScript.identifier} --action MARK_AS_EXECUTED" }
```

Comment:
The regex that should be given here is not the script identifier, it's the regex in the config. Of course, it's not very important because that's a command to execute only one script so the identifiers are not important